packages/alfa-dom/src/node/attribute/autocomplete.ts

```diff
@@ -0,0 +1,138 @@
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Parser } from "@siteimprove/alfa-parser";
+import { Slice } from "@siteimprove/alfa-slice";
+import { Array } from "@siteimprove/alfa-array";
+import { Err, Ok } from "@siteimprove/alfa-result";
+
+const { either, end, option, right, parseIf } = Parser;
+
+export namespace Autocomplete {
+
+  /**
+   * Autofill detail tokens from steps 2-4 of the list in {@link https://html.spec.whatwg.org/multipage/#autofill-detail-tokens}.
+   */
+  export namespace AutofillDetailTokens {
+    export const unmodifiables = Array.from([
```

Comment:
```suggestion
    export const unmodifiables = [
```
This is already an array, no need to turn it into one.